frontend/src/pages/Player/PlayerHook/PlayerState.ts

```diff
@@ -736,9 +748,6 @@ const processSessionMetadata = (
 		return s
 	}
 
-	if (s.replayerState < ReplayerState.Playing) {
-		s.replayerState = ReplayerState.Paused
-	}
```

Comment:
This didn't appear to be doing anything, and the `<` check may not have been accurate anymore since the `ReplayerState` enum was converted to using string values in https://github.com/highlight/highlight/pull/3308/files#diff-9e3458faf1153040ff59b48e4c06213925aaa30bd2920628945825651f76f2e9.